cns/requestcontroller/kubernetes/k8srequestcontroller.go

```diff
@@ -0,0 +1,175 @@
+package kubernetes
```

Comment:
are you thinking of having request controllers for other orchestrators? or just a test one?

Reply:
Yeah, if in the future we ever need to do this same thing with something like private link (not necessarily using CRDs but some other form of message relaying between dnc and cns)